site/docs/contribute/modules.md

```diff
@@ -2,20 +2,48 @@
 
 This page describes what must be provided when contributing a [module](../concepts/modules.md).
 
+A module is packaged as a Helm chart that the control plane can install to a workload's data plane.
+
+To first make a module available to the control plane, it must be registered by applying a FybrikModule custom resource.
+
+After that, a FybrikApplication can be submitted.  
+The FybrikApplication controller chooses the best modules to accommodate the data needs requested in the FybrikApplication, and an instace of each selected module is deployed.  
```

Comment:
requested -> declared or specified
... in the FybrikApplication, and deploys them.

Reply:
Fixed